.vscode/extensions.json

```diff
@@ -0,0 +1,8 @@
+{
+    "recommendations": [
+        "ms-vscode.cpptools",
+        "bazelbuild.vscode-bazel",
```

Comment:
Did you manage to figure out why launching builds by clicking targets is broken? If not, I'm not sure what this extension is useful for - from their [README](https://github.com/bazelbuild/vscode-bazel), I guess Bazel Build Targets tree? Linting with buildifier is already done in `fix_formatting.sh`